docker/Dockerfile

```diff
@@ -1,15 +1,13 @@
 FROM python:3.8
-WORKDIR /ci
-
+WORKDIR /wlan-testing
 RUN apt update && apt install vim -y && rm -rf /var/lib/apt/lists/*
```

Comment:
Why do we install VIM here?

Reply:
Good catch. Don't think we need it. I will remove it.